core/string/ustring.cpp

```diff
@@ -3307,6 +3307,38 @@ float String::similarity(const String &p_string) const {
 	return (2.0f * inter) / sum;
 }
 
+float String::assign_score(const String &p_base, const bool p_case_sensitive) const {
```

Comment:
```suggestion
float String::assign_score(const String &p_base, bool p_case_sensitive) const {
```

There's no need to add `const` to the primitive types, it's not doing anything, and we do not use it in our code style.